TypeError({
  name: "TypeError",
  message: "something went wrong",
  stack: "TypeError: something went wrong\n    at Object.<anonymous> (/home/ayon/Git/fron/test/errors/index.js:14:17)\n    at Module._compile (internal/modules/cjs/loader.js:1128:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)\n    at Module.load (internal/modules/cjs/loader.js:983:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:891:14)\n    at Module.require (internal/modules/cjs/loader.js:1023:19)\n    at require (internal/modules/cjs/helpers.js:72:18)\n    at Object.<anonymous> (/home/ayon/Git/fron/test/get-types.js:13:5)\n    at Module._compile (internal/modules/cjs/loader.js:1128:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)\n    at Module.load (internal/modules/cjs/loader.js:983:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:891:14)\n    at Module.require (internal/modules/cjs/loader.js:1023:19)\n    at require (internal/modules/cjs/helpers.js:72:18)\n    at /home/ayon/Git/fron/node_modules/mocha/lib/mocha.js:250:27\n    at Array.forEach (<anonymous>)\n    at Mocha.loadFiles (/home/ayon/Git/fron/node_modules/mocha/lib/mocha.js:247:14)\n    at Mocha.run (/home/ayon/Git/fron/node_modules/mocha/lib/mocha.js:576:10)\n    at Object.<anonymous> (/home/ayon/Git/fron/node_modules/mocha/bin/_mocha:637:18)\n    at Module._compile (internal/modules/cjs/loader.js:1128:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)\n    at Module.load (internal/modules/cjs/loader.js:983:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:891:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"
})